src/dmqtest/DmqTestCase.d

```diff
@@ -833,8 +833,7 @@ class SubscribeDmqTestCase : DmqTestCase
 
         foreach (channel, records_by_subscriber; records_by_channel_and_subscriber)
             foreach (subscriber, records; records_by_subscriber)
-                foreach (i, record; records.sort)
+                foreach (i, record; sort(records))
```

Comment:
Why not just use https://github.com/sociomantic-tsunami/ocean/blob/v3.x.x/src/ocean/core/array/Mutation.d#L561?

Reply:
It causes a stack overflow due to excessive recursion. I added a comment to my custom `sort` function..